src/main/java/frc/robot/OI.java

```diff
@@ -44,39 +85,96 @@ public static Supplier<Double> getRotationSupplier(){
         }
 
         public static JoystickButton getOrientationButton(){
-            return new JoystickButton(kJoystick, kOrientationButtonID);
+            kOrientationButton.setButtonName("Orientation Button");
+            return new JoystickButton(kJoystick, kOrientationButton.getButtonID());
         }
 
         public static JoystickButton getZeroButton(){
-            return new JoystickButton(kJoystick, kZeroButtonID);
+            kZeroButton.setButtonName("Zero Button");
+            return new JoystickButton(kJoystick, kZeroButton.getButtonID());
         }
 
         public static JoystickButton getAlignForwardButton(){
-            return new JoystickButton(kJoystick, kAlignForwardButtonID);
+            kAlignForwardButton.setButtonName("Align Forward");
+            return new JoystickButton(kJoystick, kAlignForwardButton.getButtonID());
         }
 
         public static JoystickButton getAlignBackButton(){
-            return new JoystickButton(kJoystick, kAlignBackwardButtonID);
+            kAlignBackwardButton.setButtonName("Align Backward");
+            return new JoystickButton(kJoystick, kAlignBackwardButton.getButtonID());
         }
 
         public static JoystickButton getIntakeButton(){
-            return new JoystickButton(kJoystick, kIntakeButtonID);
+            kIntakeButton.setButtonName("Intake Button");
+            return new JoystickButton(kJoystick, kIntakeButton.getButtonID());
         }
         
         public static JoystickButton getOuttakeButton(){
-            return new JoystickButton(kJoystick, kOuttakeButtonID);
+            kOuttakeButton.setButtonName("Outtake Button");
+            return new JoystickButton(kJoystick, kOuttakeButton.getButtonID());
         }
     }
 
     public static final class Operator{
+        public enum OperatorButtons {
+            X (1, null),
+            A (2, null),
+            B (3, null),
+            Y (4, null),
+            LB (5, null),
+            RB (6, null),
+            LT (7, null),
+            RT (8, null),
+            Back (9, null),
+            Start (10, null),
+            LJ (11, null),
+            RJ (12, null);
+            
+            private final int buttonID; 
+            private String buttonName;
+          
+            OperatorButtons(int ID, String N) {
```

Comment:
May as well leave String N out of the parameter list and assign buttonName to a default value like "No assignment".